### ISSUE TITLE
`changedAny` and `changedOnly` functions

### ISSUE THREAD

**Issue by saiichihashimoto** [sanity-io/groq-js#160]
The [groq](https://sanity-io.github.io/GROQ/GROQ-1.revision1/#sec-Diff-namespace) [spec](https://sanity-io.github.io/GROQ/GROQ-1.revision1/#sec-Delta-namespace) and the [sanity docs](https://www.sanity.io/docs/groq-functions#8fbcf5335cbc) both mention the `changedAny()` and `changedOnly()` functions, but groq-js doesn't support them yet.

**Comment by judofyr** [comment]
Yup, it would be nice to have these. It's just been deprioritized since they are rarely used in queries on the client-side.
